modular_skyrat/master_files/code/modules/health/health_mods.dm

```diff
@@ -0,0 +1,74 @@
+#define PAIN_CAP 500
+#define PAIN_MIN 0
+#define SHOCK_THRESH 50
+#define PAIN_WOUND_MINOR 30
+#define PAIN_WOUND_MODERATE 50
+#define PAIN_WOUND_MAJOR 80
+#define MAGIC_PAIN_DIVISOR 10
```

Comment:
could you write a comment or change the variable name to better explain how this works?